discovery-provider/plugins/notifications/src/processNotifications/mappers/follow.ts

```diff
@@ -55,6 +56,11 @@ export class Follow extends BaseNotification<FollowNotificationRow> {
       this.identityDB,
       [this.receiverUserId, this.followerUserId]
     )
+
+    const title = 'Follow'
```

Comment:
oo while you're here mind changing this title to `new follow` so we can knock [this ticket](https://linear.app/audius/issue/PLAT-902/[plugin]-push-notification-for-follow-should-say-new-follow) out too ?

Reply:
totally!